ui/cypress/e2e/buckets.test.ts

```diff
@@ -99,6 +99,24 @@ describe('Buckets', () => {
             })
           })
 
+        cy.getByTestID('resource-sorter--button')
+          .click()
+          .then(() => {
+            cy.getByTestID(
+              'resource-sorter--retentionRules[0].everySeconds-asc'
+            ).click()
+          })
+          .then(() => {
+            const asc_buckets = buckets
+              .slice()
+              .sort()
```

Comment:
`Array.prototype.sort` does not sort lexicographically.  to guarantee the alphabetical ordering of these strings you can do: 

```
arr.sort((a, b) =>  a.localeCompare(b))
```

Reply:
Got it! i also adjusted the other sort function to match this style :)